app/models/user.rb

```diff
@@ -10,14 +10,16 @@ class User < ApplicationRecord
   has_secure_password
   validates :password, presence: true, length: { minimum: Settings.validates.password.minimum }
 
-  def User.digest(string)
-    cost = ActiveModel::SecurePassword.min_cost ? BCrypt::Engine::MIN_COST :
-      BCrypt::Engine.cost
-    BCrypt::Password.create(string, cost: cost)
-  end
+  class << self
+    def digest string
+      cost = ActiveModel::SecurePassword.min_cost ? BCrypt::Engine::MIN_COST :
+        BCrypt::Engine.cost
+      BCrypt::Password.create(string, cost: cost)
```

Comment:
remove ()

Reply:
Yes,bro
I see 2 paragrams, so I not remove :))